src/interpreter/InterpretAdditionalItems.ts

```diff
@@ -22,8 +22,12 @@ export default function interpretAdditionalItems(
   if (typeof schema === 'boolean' || model.type?.includes('array') === false) {
     return;
   }
+  const ignoreAdditionalItems =
+    interpreterOptions.ignoreAdditionalItems || false;
   const additionalItemsModel = interpreter.interpret(
-    schema.additionalItems === undefined ? true : schema.additionalItems,
+    schema.additionalItems === undefined
+      ? !ignoreAdditionalItems
+      : schema.additionalItems,
```

Comment:
In order to not overcomplicate stuff, how about we just ignore the interpretation instead of interpreting it with `false` 🤔 ? 
```suggestion
  const ignoreAdditionalItems =
    interpreterOptions.ignoreAdditionalItems || false;
  if (typeof schema === 'boolean' || model.type?.includes('array') === false || ignoreAdditionalItems) {
    return;
  }
  const additionalItemsModel = interpreter.interpret(
    schema.additionalItems === undefined ? true : schema.additionalItems,
```

Reply:
The logic has been simplified.